conn.go

```diff
@@ -116,6 +116,10 @@ func messageUnmarshall(msgs []netlink.Message) ([]Message, []netlink.Message, er
 			m = &AddressMessage{}
 		case RTM_NEWROUTE:
 			m = &RouteMessage{}
+		case RTM_DELROUTE:
+			fallthrough
+		case RTM_GETROUTE:
+			fallthrough
 		default:
```

Comment:
The fallthrough should be above the `case RTM_NEWROUTE` for it to function properly.